var/spack/repos/builtin/packages/axom/package.py

```diff
@@ -620,34 +621,27 @@ def build_test(self):
 
     @run_after("install")
     @on_package_attributes(run_tests=True)
-    def check_install(self):
-        """
-        Checks the spack install of axom using axom's
-        using-with-cmake example
-        """
-
-        print("Checking Axom installation...")
-        spec = self.spec
-        install_prefix = spec.prefix
-        example_src_dir = join_path(install_prefix, "examples", "axom", "using-with-cmake")
-        example_build_dir = join_path(example_src_dir, "build")
-        print("Checking using-with-cmake example...")
-        with working_dir(example_build_dir, create=True):
+    def test_install_using_cmake(self):
+        """build example with cmake and run"""
+        example_src_dir = join_path(self.prefix.examples.axom, "using-with-cmake")
+        shutil.copytree(example_src_dir, "./cmake")
+        with working_dir("./cmake/build", create=True):
```

Comment:
```suggestion
        example_src_dir = join_path(self.prefix.examples.axom, "using-with-cmake")
        example_stage_dir = "./cmake"
        shutil.copytree(example_src_dir, example_stage_dir)
        with working_dir(join_path(example_stage_dir, "build"), create=True):
```
Maybe it is because the coffee hasn't kicked in but the `./cmake` where did not immediately stick out that it was inside the stage directory. I think this is more clear. If you agree could you do it to the make example as well?

Reply:
Done.  Added the variable to both locations.